### DISCUSSION TITLE
Add an up-select attribute, to select a fragment from the response via CSS selector.

### DISCUSSION THREAD

**Discussion by panda-madness** [unpoly/unpoly#661] [Ideas]
This would work pretty much like [hx-select](https://htmx.org/attributes/hx-select/) does in HTMX.

**Comment by triskweline** [comment]
You can use the [`[up-target]`](https://unpoly.com/a-up-follow#up-target) attribute to select the fragment to update.

**Reply by panda-madness** [reply]
Up-target requires that both the source page and the response both contain the selector provided. What i propose is that an up-select attribute could allow us to select a different chunk just from the response.
So a link with [up-target=#mytarget up-select=#mydiv] would insert the contents of #mydiv from the response and put it in #mytarget in the source page.
